src/features/ui/Switch/Switch.tsx

```diff
@@ -0,0 +1,43 @@
+import { FC } from 'react';
```

Comment:
We should add this one to `zUI`

Reply:
For sure, there are a number of components within `zApp-NFTs` that need to be pulled over to `zUI`. If your happy to leave here for now, we can add a task to the back log to focus on actioning these?

Reply:
Sounds good, create a backlog task for it and we can do it next sprint or whenever it comes up!

Reply:
Added to backlog :)